experimental/framework/frs-client/src/FrsAzFunctionTokenProvider.ts

```diff
@@ -10,7 +10,7 @@ import { FrsMember } from "./interfaces";
 export class FrsAzFunctionTokenProvider implements ITokenProvider {
     constructor(
         private readonly azFunctionUrl: string,
-        private readonly user?: Pick<FrsMember, "userId" | "userName">,
+        private readonly user?: Pick<FrsMember<any>, "userId" | "userName" | "additionalDetails">,
```

Comment:
If we use the `T=any` default value in FrsMember, we can omit the `any` here
```suggestion
        private readonly user?: Pick<FrsMember, "userId" | "userName" | "additionalDetails">,
```